### PR TITLE
update to v7.3.5, default to python3.7.10, add win64

### PR DESCRIPTION
**Pull request by mattip**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattip** [comment]
@conda-forge-admin, please rerender

**Comment by mattip** [comment]
Fixes #16, #15

File: recipe/meta.yaml [hunk 1]
```diff
@@ -76,7 +69,7 @@ outputs:
         - test $(python -c "import cffi;print(cffi.__version__)") == "{{ cffi_version }}"
 
 about:
-  home: https://github.com/conda-forge/pypy-feedstock
+  home: https://github.com/conda-forge/pypy-meta-feedstock
```
Comment (mattip) [inline recipe/meta.yaml:72]: Is this right?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -31,11 +29,6 @@ outputs:
           - python_abi {{ python_maj_min }} *_pypy{{ python_maj_min.replace(".", "") }}_pp{{ pypy_abi }}
         noarch:
           - {{ pin_subpackage("python", max_pin="x.x", min_pin="x.x") }}
-      script:
-        - ln -sf ${PREFIX}/bin/pypy3 ${PREFIX}/bin/python
-        - ln -sf ${PREFIX}/bin/pypy3 ${PREFIX}/bin/python3
-        - ln -sf ${PREFIX}/bin/pypy3 ${PREFIX}/bin/python{{ python_maj_min }}
```
Comment (isuruf) [inline recipe/meta.yaml:37]: This symlink is not there in `pypy3.7` package right?

File: recipe/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipe/meta.yaml:72]: Yes

File: recipe/meta.yaml [hunk 3]
```diff
@@ -31,11 +29,6 @@ outputs:
...
-        - ln -sf ${PREFIX}/bin/pypy3 ${PREFIX}/bin/pypy
```
Comment (mattip) [inline recipe/meta.yaml:38]: Now done in the packaging step as part of pypy3.6-feedstock calling `python .../package.py`

Comment (isuruf) [reply recipe/meta.yaml:38]: Got it.

File: recipe/meta.yaml (hunk 2 above)
Comment (mattip) [reply recipe/meta.yaml:37]: It was added in response to this repo [here](https://foss.heptapod.net/pypy/pypy/-/blob/release-pypy3.7-v7.3.5/pypy/tool/release/package.py#L252) in [a commit](https://foss.heptapod.net/pypy/pypy/-/commit/bc470c4a1fe5c9e082842e7123369b8aa3866577) six months ago

**Comment by mattip** [comment]
I can understand why aarch64 is failing: I didn't trigger a build in pypy3.6-feedstock (same for travis). I am a bit lost with the windows failures.

**Comment by isuruf** [comment]
For windows, we need to update https://github.com/conda-forge/python_abi-feedstock/blob/master/recipe/conda_build_config.yaml too.

**Comment by h-vetinari** [comment]
@conda-forge-admin, please restart ci

**Comment by h-vetinari** [comment]
Ah, right, drone doesn't pick up restarts. Sounds like someone needs to push a commit.

**Comment by mattip** [comment]
How does pyyp3.6-feedstock get non-x86 hardware builds?

**Comment by mattip** [comment]
Should I push an empty commit to trigger the drone build?

**Comment by mattip** [comment]
@conda-forge-admin, please rerender

**Comment by isuruf** [comment]
Yes, an empty commit is the way to go.

**Comment by mattip** [comment]
Cool. Is there something else before conda-forge can roll out the win64 migration of packages available for pypy3?

**Comment by h-vetinari** [comment]
I would have this interpreted this PR as updating the pypy release to 7.3.5, but conda-forge CI is still pulling in 7.3.4, e.g. [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=344167&view=logs&j=517fe804-fa30-5dc2-1413-330699242c05&t=c10fa5f2-fdf6-5338-3bdb-c4bea7c23412) (from [this](https://github.com/conda-forge/numpy-feedstock/pull/227) PR):
```
    pypy3.7:            7.3.4-hc4e864a_4         conda-forge
    python:             3.7.10-0_73_pypy         conda-forge
    python_abi:         3.7-2_pypy37_pp73        conda-forge
```

Which is doubly weird, because it uses the `python_version` as set in this PR (3.7.10) while ignoring that pypy3.7 should be set to 7.3.5

**Comment by isuruf** [comment]
> Which is doubly weird, because it uses the python_version as set in this PR (3.7.10) while ignoring that pypy3.7 should be set to 7.3.5

That's not true. pypy3.7 7.3.4 and 7.3.5 both provide python 3.7.10.

What we need is a build number bump here.

**Pull request merged**